README.md

```diff
@@ -1,33 +1,151 @@
+[![banner](examples/tutorials/quick-start/media/PF_banner.png)](https://microsoft.github.io/promptflow/index.html)
+
+------
+
 # Prompt flow
 
 [![Python package](https://img.shields.io/pypi/v/promptflow)](https://pypi.org/project/promptflow/)
-[![Python](https://img.shields.io/pypi/pyversions/promptflow.svg?maxAge=2592000)](https://pypi.python.org/pypi/promptflow/) 
+[![CLI](https://img.shields.io/badge/CLI-reference-blue)](https://microsoft.github.io/promptflow/reference/pf-command-reference.html)
+[![Doc](https://img.shields.io/badge/Doc-online-green)](https://microsoft.github.io/promptflow/index.html)
 [![PyPI - Downloads](https://img.shields.io/pypi/dm/promptflow)](https://pypi.org/project/promptflow/)
+[![Issue](https://img.shields.io/github/issues/microsoft/promptflow)](https://github.com/microsoft/promptflow/issues/new/choose)
+[![Discord](https://dcbadge.vercel.app/api/server/bnXr6kxs?compact=true&style=flat)](https://discord.gg/bnXr6kxs)
+[![Discussions](https://img.shields.io/github/discussions/microsoft/promptflow)](https://github.com/microsoft/promptflow/issues/new/choose)
+[![CONTRIBUTING](https://img.shields.io/badge/Contributing-8A2BE2)](https://github.com/microsoft/promptflow/blob/main/CONTRIBUTING.md)
 [![License: MIT](https://img.shields.io/github/license/microsoft/promptflow)](https://github.com/microsoft/promptflow/blob/main/LICENSE)
 
-> Welcome to join us to make Prompt flow better by
-> participating [discussions](https://github.com/microsoft/promptflow/discussions),
-> opening [issues](https://github.com/microsoft/promptflow/issues/new/choose),
-> submitting [PRs](https://github.com/microsoft/promptflow/pulls).
+> Welcome to join us to make Prompt flow!
 
 **Prompt flow** is a suite of development tools designed to streamline the end-to-end development cycle of LLM-based AI applications, from ideation, prototyping, testing, evaluation to production deployment and monitoring. It makes prompt engineering much easier and enables you to build LLM apps with production quality.
 
 With prompt flow, you will be able to:
 
-- Create executable workflows that link LLMs, prompts, Python code and other tools together.
-- Debug and iterate your flows, especially the interaction with LLMs with ease.
-- Evaluate your flow's quality and performance with larger datasets.
-- Integrate the testing and evaluation into your CI/CD system to ensure quality of your flow.
-- Deploy your flow to the serving platform you choose or integrate into your app's code base easily.
-- (Optional but highly recommended) Collaborate with your team by leveraging the cloud version of [Prompt flow in Azure AI](https://learn.microsoft.com/en-us/azure/machine-learning/prompt-flow/overview-what-is-prompt-flow?view=azureml-api-2).
+- **Create and Iteratively Develop Flow**
```

Comment:
may create some link to concept doc here, like flow

Reply:
added